tests/bluetooth/tester/src/gap.c

```diff
@@ -825,6 +825,19 @@ enum bt_security_err auth_pairing_accept(struct bt_conn *conn,
 	return BT_SECURITY_ERR_SUCCESS;
 }
 
+void auth_pairing_failed(struct bt_conn *conn, enum bt_security_err reason)
+{
+	struct gap_bond_pairing_failed_ev ev;
+	const bt_addr_le_t *addr = bt_conn_get_dst(conn);
+
+	memcpy(ev.address, addr->a.val, sizeof(ev.address));
```

Comment:
Minor: Typically when ignoring return values, we cast to void:
```suggestion
	(void)memcpy(ev.address, addr->a.val, sizeof(ev.address));
```

Reply:
@sjanc - please update